fixtures/input.txt

```diff
@@ -3,7 +3,7 @@ To: foo@example.com, bar@example.com, baz@example.com, quz@example.com, foobar@e
 	foobuzz@example.com, fooquz@example.com, bonk@example.com, bink@example.com, bank@example.com, bossman@example.com
 
 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-
+ 
```

Comment:
No, we're ignoring the whitespace and treating it like a newline so there's no difference in the output. But without the changes, adding this whitespace causes the newline between the paragraphs to be removed entirely.